components/layout/DocsLayout.js

```diff
@@ -142,11 +142,11 @@ export default function DocsLayout({ post, navItems = {}, children }) {
               </SearchButton>
               
               <nav className="flex-1 bg-white">
-                <ul>
+                <u l>
```

Comment:
This change is quite confusing to me because, in this way, I think you are allowing to have dark points beside each Nav option in Docs and it is not correct in terms of my view. I'll still ask for opinions from @derberg and @alequetzalli.

![image](https://user-images.githubusercontent.com/76521428/200585739-fa25a1bd-c9c2-4fb6-af85-3e8baf658fd1.png)

Reply:
Agreed, we should NOT have <li> bullet points showing.

Reply:
Fixed!